src/rss-diff.js

```diff
@@ -1,5 +1,25 @@
-function rssNDiff(itemBefore, itemAfter, field) {
-    let delta
+function rssDiff(itemBefore, itemAfter, field) {
+    let beforeMap = {}
+    let out = []
+    let indicies = []
+
+    // Create a map of field to index for first array
+    for (let i = 0; i < itemBefore.length; i++) {
+        beforeMap[itemBefore[i][field]] = i
```

Comment:
I think it would be better to just set each of these to `true` or `1` instead of the index unless you plan on using the index.